CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,8 @@
+<!--
+SPDX-FileCopyrightText: © 2020 fish-shell contributors
+
+SPDX-License-Identifier: GPL-2.0-only
```

Comment:
This is the contributor covenant, copied from https://www.contributor-covenant.org/version/2/1/code_of_conduct/ (I'm not sure which version).

It carries a CC-BY 4.0 license

Reply:
Then perhaps something like this instead:

```suggestion
SPDX-License-Identifier: CC-BY-4.0
```